src/cse.cpp

```diff
@@ -3,7 +3,7 @@
 // that can be found in the LICENSE file.
 
 ///////////////////////////////////////////////////////////////////////////////
-// CSE.cpp : Main program module
+// cse.cpp : Main program module
```

Comment:
In our last iteration meeting, we touched upon the fact that the CSE executable name is in all caps, and maybe would remain that way. Does that argue for cse.h and cse.cpp to stay capitalized too? Obviously we can have CMake call the exe whatever we want, but in a simple build the file with `main()` would be the name of the binary.